certificate-manager/src/main/java/io/strimzi/certs/SecretCertProvider.java

```diff
@@ -55,17 +69,33 @@ public Secret createSecret(String namespace, String name, File keyFile, File cer
      * @param certKey key field in the Secret data section for the certificate
      * @param keyFile private key to store
      * @param certFile certificate to store
+     * @param storeKey key field in the Secret data section for the PKCS12 store
+     * @param storePasswordKey key field in the Secret data section for the PKCS12 store password
+     * @param storeFile PKCS12 store
+     * @param storePassword PKCS12 store password
      * @param labels Labels to add to the Secret
      * @param annotations annotations to add to the Secret
      * @param ownerReference owner of the Secret
      * @return the Secret
      * @throws IOException If a file could not be read.
      */
-    public Secret createSecret(String namespace, String name, String keyKey, String certKey, File keyFile, File certFile, Map<String, String> labels, Map<String, String> annotations, OwnerReference ownerReference) throws IOException {
+    public Secret createSecret(String namespace, String name,
+                               String keyKey, String certKey,
+                               File keyFile, File certFile,
+                               String storeKey, String storePasswordKey,
```

Comment:
Same here - can we clarify what is keystore and what is truststore? (And keep in mind that we will probably need to add the other later?)

Reply:
About using "store" I could say the same as the comment above about using "store" in the CertAndKey class.